### ISSUE TITLE
Does this output for vanilla HTML?

### ISSUE THREAD

**Issue by Autr** [sinedied/svelte-web-components-template#1]
Hello, just testing this out, but noticed the demo app is using a compiled Svelte bundle and App.svelte - after building and loading dist/index.js into a plain HTML file console logs `Uncaught ReferenceError: module is not defined` on line 1 - is there an example of the web-component loading without Svelte? Thanks!

**Comment by sinedied** [comment]
That's intended, the demo app is there to help with the development so you have somewhere to test and look at your component, but it's a svelte app you're right.

Once the component library is built, it produces vanilla html/js web components that you can import in any project. There's currently no example but that could be something added to the readme.
You usually need nothing more than:
```html
<!-- import the lib -->
<script type="module" src="my-lib.js"></script>
<!-- use the components -->
<my-custom-element></my-custom-element>
```

**Comment by Autr** [comment]
Hi @sinedied - yep! The only problem I had was that my component hadn't compiled properly - switching back to your example I saw it was working and refactored step-by-step from here. 

In terms of dev workflow I removed the App.svelte example and added `spawn node start` callback inside rollup.config.js to launch a basic http server on a vanilla HTML file that imports dist output and tests the web-components. Example is here: https://github.com/autr/ezelte

Might be nice to add something similar to the template project - save the confusion between svelte app / web-component?
